### PR TITLE
start

### PR DESCRIPTION
**Pull request by farzod-achilov**
start

### CODE REVIEW

File: scss/_common.scss [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+  box-sizing: border-box;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
 }
 
 .container {
-    max-width: 1200px;
-    margin: 0 auto;
+  max-width: 1200px;
```
Comment (farzod-achilov) [inline scss/_common.scss]: container 1500 bo`lishi kerak

Comment (farzod-achilov) [reply scss/_common.scss]: 1280 ekan

**Comment by Shoha-N1** [comment]
add new branch

**Comment by farzod-achilov** [comment]
Fine

**Comment by farzod-achilov** [comment]
123

**Pull request closed without merging**